ui/src/app/applications/components/application-details/application-details.tsx

```diff
@@ -186,122 +188,185 @@ export class ApplicationDetails extends React.Component<RouteComponentProps<{nam
+                                                                            }}
+                                                                        />
+                                                                        <i
+                                                                            className={classNames('fa fa-network-wired', {selected: pref.view === 'network'})}
+                                                                            title='Network'
+                                                                            onClick={() => {
+                                                                                this.appContext.apis.navigation.goto('.', {view: 'network'}, {replace: true});
+                                                                                services.viewPreferences.updatePreferences({appDetails: {...pref, view: 'network'}});
+                                                                            }}
+                                                                        />
+                                                                        <i
+                                                                            className={classNames('fa fa-th-list', {selected: pref.view === 'list'})}
+                                                                            title='List'
+                                                                            onClick={() => {
+                                                                                this.appContext.apis.navigation.goto('.', {view: 'list'}, {replace: true});
+                                                                                services.viewPreferences.updatePreferences({appDetails: {...pref, view: 'list'}});
+                                                                            }}
+                                                                        />
+                                                                    </div>
+                                                                </React.Fragment>
                                                             )
-                                                        }
-                                                        quickStarts={node => AppUtils.renderResourceButtons(node, application, tree, this.appContext, this.appChanged)}
-                                                    />
-                                                )) || (
-                                                    <div>
-                                                        <Filters pref={pref} tree={tree} onSetFilter={setFilter} onClearFilter={clearFilter}>
-                                                            {(filteredRes.length > 0 && (
+                                                        }}>
+                                                        <div className='application-details__application-search-bar'>
```

Comment:
The "refreshing" label is now hidden by the additional "toolbar" panel. Can we consider re-using page title instead of introducing new toolbar? E.g.

![image](https://user-images.githubusercontent.com/426437/148840488-e4d0ed7a-a6b2-48a1-ad31-f574a660bd95.png)

Reply:
Hey, sorry to jump into this discussion, from my point of view its a lot "prettier" to introduce the search bar, but either solution is okay as long as the feature is implemented because it is a must :)